### ISSUE TITLE
Please release Rails 5 version

### ISSUE THREAD

**Issue by Fuffi** [evendis/mandrill-rails#46]
Is this gem completely unsupported? Would be nice to get a Rails-5 ready version, there are lots of forks that do that already

**Comment by tardate** [comment]
thanks everyone for bugging me about this! I just merged the various efforts from @prosanelli @shanna @PikachuEXE @Fuffi @spastorino @Flarra @tomdracz on #43 #42 #36 and done some cleanup to get a green ci. It is merged on master but I haven't released a gem yet.

Problem is I don't run mandrill with any rails 5 apps yet, so my confidence this actually works is only as good as the unit-like testing. can someone does us all a favour and test this out for real and give a thumbs up/down?

**Comment by Fuffi** [comment]
I will update the gem and try it at some point, thanks a lot!

**Comment by tomdracz** [comment]
Hey there, @tardate Thanks for merges!
I've been running my fork in production (Rails 5.1) for close to two weeks and so far so good, no errors encountered.
Think we've got everything we need merged namely:
- change the _filter to _action
- add raise: false to callbacks
- change setup in tests

If I encounter anything, I'll comment here but I think it's good to go.

One thing to note, dropping _filter callbacks meant we've dropped support for Rails below version 4. Might be worth mentioning in the readme that people still running those should use 1.4.1 version of gem

**Comment by tomdracz** [comment]
@tardate Actually looking at Gemspec, we dropped anything below Rails 5. Are there any issues with CI if we use active support >= 4 as dependency? Think that was done in one of the pull requests but has been changed since to depend on version >= 5
One way or another, we can just use current version as Rails 5 and advise users of previous ones to point gemfiles at version 1.4.1

**Comment by tardate** [comment]
@tomdracz thanks for the feedback, yes we can probably wind back the active support dependency, depending on when the `_filter` to `_action` change first appeared. Can anyone help with a definitive version that should be in the dependencies?

NB: I've added the comment in the readme to point non-rails 5 users to 1.4.1, I think that's fine for now. @spastorino had some suggestions for backwards compatibility on #42, but I've left that as a "todo" for anyone keen to help make that a reality.

**Comment by tomdracz** [comment]
@tardate Looks like before_action first appeared in Rails 4 so putting active support at >= 4 should work.
Think the backwards compatibility might be neat, but if we point users to 1.4.1, it should be enough. Bundler will warn about dependencies mismatch and 1.4.1 will be the only one they can use.

Actually Rails team has been discussing brining back before_filter as aliases here: https://github.com/rails/rails/issues/29030 but the consensus is that the ship has sailed. Think we should move along here also

**Comment by tardate** [comment]
thanks folks. I dropped the requirement back to >= 4 and published 1.5.0. Raise more issues as they arise!

```
$ gem list -r mandrill-rails

*** REMOTE GEMS ***

mandrill-rails (1.5.0)
```
